newsfragments/2268.feature

```diff
@@ -0,0 +1 @@
+``dials.refine``: use an overall ``interval_width_degrees`` parameter to set defaults for all models simultaneously.
```

Comment:
Does this set it for _all_ models?

For example I would _not_ expect 

```
refinement.parameterisation.detector.smoother.interval_width_degrees
```

to be affected?

Reply:
As in: I had expected that this would affect all _crystal_ models

Reply:
With an additional sentence explaining the default behaviour here as discussed in the conversation for the PR looks great, thank you.